platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -206,6 +220,33 @@ public Path getExportFile() {
     return contexts.values();
   }
 
+  private Map<String, ?> getSystemProperties(Map<String, Object> metadata) {
+    if (version.equals(MigrationContextImpl.SYSTEM_PROPERTIES_MISSING_VERSION)) {
+      return AccessUtils.doPrivileged(
+          () -> {
+            ImportMigrationContext platformMigratable =
+                contexts.get(MigrationContextImpl.PLATFORM_MIGRATABLE_ID);
+            try {
+              URL systemPropsURL =
+                  new URL(
+                      null,
+                      MigrationURLConnection.SYSTEM_PROPERTIES_URL,
+                      new URLStreamHandler() {
+                        @Override
+                        protected URLConnection openConnection(URL u) throws IOException {
+                          return new MigrationURLConnection(u, platformMigratable);
+                        }
+                      });
+              return PropertiesLoader.loadPropertiesFile(systemPropsURL, true);
```

Comment:
I wasn't sure what the best way to test getting the system properties from the platform migratable was, since a lot of this is internal to the class. I tried to manually create the zip entry so that the import migration manager would create the entry that was needed, but I couldn't figure out the correct structure. If there is a good way to test this, please let me know!